Makefile

```diff
@@ -90,6 +90,6 @@ clean:
 # Generate environment
 .PHONY: env
 env: 
-	@export $(find . -name '*.env' -exec grep -rhv -e '^#' -e '^$' {} + | xargs)
-	@export MORPH_BLOCK_TIME=$(grep 'SecondsPerBlock' $(MORPH_CHAIN_PROTOCOL) | tail -n 1 | cut -d: -f2) 
-	@export MAINNET_BLOCK_TIME=$(grep 'SecondsPerBlock' $(CHAIN_PROTOCOL) | tail -n 1 | cut -d: -f2)
+	@echo $(find . -name '*.env' -exec grep -rhv -e '^#' -e '^$' {} + | xargs)
```

Comment:
This find statement also has issues.

1. Add `shell` keyword
2. `$` needs to be escaped as in `print $$2` 
3. Is `xargs` required there?

Reply:
3. Not sure... I may remove it if you think it's not required.